src/main/java/gift/controller/KakaoApiController.java

```diff
@@ -0,0 +1,45 @@
+package gift.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import gift.auth.KakaoAuth;
+import gift.dto.KakaoTokenResponseDto;
+import gift.dto.ProductResponseDto;
+import gift.service.KakaoApiService;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+
+@RestController
+@RequestMapping("/api/kakao")
+public class KakaoApiController {
+
+    private final KakaoApiService kakaoApiService;
+    private String accessToken = null;
+
+    public KakaoApiController(KakaoApiService kakaoApiService) {
+        this.kakaoApiService = kakaoApiService;
+    }
+
+    @GetMapping
+    public ResponseEntity<KakaoTokenResponseDto> getAccessToken() {
+        KakaoTokenResponseDto responseDto = kakaoApiService.getAccessToken();
+        accessToken = responseDto.getAccess_token();
+        return ResponseEntity.status(HttpStatus.OK).body(responseDto);
```

Comment:
아래와 같이 간단하게 변경해볼 수 있을 것 같아요. 😃 
```suggestion
        return ResponseEntity.ok(responseDto);
```

Reply:
개인적으로는 기존 방법이 보기 편한거 같은데 이것도 컨벤션 같은 개발자들의 암묵적 룰 같은게 있을까요?